### PR TITLE
Solves ambiguous usage of named arguments in PHP 8

### PR DESCRIPTION
**Pull request by dgafka**
Screenshot from 2020-10-15 15-52-05

### CODE REVIEW

**Comment by ciaranmcnulty** [comment]
As this isn't picked up by the tests, can you add a test that would trigger the behaviour previously?

File: src/Behat/Testwork/Call/Handler/RuntimeCallHandler.php [hunk 1]
```diff
@@ -105,6 +105,7 @@ private function executeCall(Call $call)
         $return = $exception = null;
 
         try {
+            $arguments = array_values($arguments);
             $this->validator->validateArguments($reflection, $arguments);
             $return = $callable(...array_values($arguments));
```
Comment (stof) [inline src/Behat/Testwork/Call/Handler/RuntimeCallHandler.php:110]: This `array_values` can be removed now that you moved it earlier (but is it actually an issue in `validateArguments` or are you using a Behat version which does not have this `array_values` yet ?)

**Comment by dgafka** [comment]
> As this isn't picked up by the tests, can you add a test that would trigger the behaviour previously?

If you set up testing for PHP 8, I think a lot of existing tests will fail. 
With current setup for <= 7.4 there is no way to show this issue, as it's related to PHP 8 functionality, named parameters.

File: src/Behat/Testwork/Call/Handler/RuntimeCallHandler.php (hunk 1 above)
Comment (dgafka) [reply src/Behat/Testwork/Call/Handler/RuntimeCallHandler.php]: Good catch, I have removed duplicated call :)

I am not sure, where this issue comes from I could not track it all the way. 
However I am pretty sure, you will see the issue, when test suite will be set up for PHP 8.

**Comment by ciaranmcnulty** [comment]
The reason I ask about a test is that #1313 I believe fixes all the outstanding failures under PHP 8 so we are probably missing coverage of this case

**Comment by ciaranmcnulty** [comment]
Any idea why the build is failing?

**Comment by ciaranmcnulty** [comment]
It looks like the test was just sensitive to the line numbers in a stack trace (that changed because you added a line)

**Comment by ciaranmcnulty** [comment]
Thanks @dgafka !

**Pull request merged**